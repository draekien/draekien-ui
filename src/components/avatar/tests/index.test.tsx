import { render } from '@testing-library/react';
import React from 'react';
import { Avatar } from '../index';

describe('Avatar component', () => {
  test('should match snapshot', () => {
    const { container } = render(<Avatar />);

    expect(container).toMatchSnapshot();
  });
});
