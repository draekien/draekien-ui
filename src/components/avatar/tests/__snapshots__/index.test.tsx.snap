// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component should match snapshot 1`] = `
<div>
  <div
    class="css-1tvgq5r-Avatar"
  >
    <img
      class="css-xrh3x4-Avatar"
      src="https://avatars.dicebear.com/api/jdenticon/default.svg"
    />
  </div>
</div>
`;
